pages/checkout/index.tsx

```
@@ -0,0 +1,422 @@
+                setStatus('processing');
+                break;
+
+              case 'checkout.closed':
+                // User closed the overlay - go back to ready state
+                if (status !== 'succeeded' && status !== 'processing') {
+                  setStatus('ready');
+                }
+                break;
+
+              case 'checkout.status':
+                const statusData = event.data?.message as { status?: string };
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)